source/external/android-specific/README.md

```diff
@@ -0,0 +1,5 @@
+## source
+
+Repo: https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin
```

Comment:
+1
Let's do this for all libraries

Reply:
Right, when one developer add a library. It's better to add a `README.md` to explain where it comes